deploy-sentinel/teletraan/STOPPING

```diff
@@ -0,0 +1,2 @@
+
+echo Running STOPPING script.
```

Comment:
you may want to add a long sleep here so you can ssh into the host to see the log.   as the log will only available on the host.

Reply:
done.

Reply:
I think having the echo before sleep makes more sense.

Reply:
I believe the purpose of having sleep is to give time to ssh into the system, else we might never see the echo in the console. Correct me if I am wrong

Reply:
agree with Tyler that sleep after echo is better.  the output of the script will be stored in /var/log/deployd/<service>.log, so after you ssh in, you can check the output during the sleep period.  otherwise, the script will exit and the host will be terminated after the script exit.    another suggestion is to increase the 5min to 10min so you have enough time.